app/views/distributions/new.html.erb

```diff
@@ -49,6 +49,9 @@
 
                   <%= f.input :issued_at, as: :datetime, ampm: true, minute_step: 15, label: "Distribution date" %>
                   <%= f.input :reminder_email_enabled, as: :boolean, checked_value: true, unchecked_value: false, label: "Send email reminder the day before?" %>
+                  <%= f.input :issued_at_timeframe_enabled, as: :boolean, checked_value: true, unchecked_value: false, label: "Enable time frame?" %>
```

Comment:
Let's make this `f.checkbox` too for consistency

Reply:
Just updated the form for consistency as requested. Also, I did a check of the mailer template to make sure changes haven't brokend anything. I noticed that the regular confirmation email did not include the pick-up time even though a user selects a pickup time when creating a distribution. So I edited the email template to include the pickup time selected. I'm attaching a screenshot of this below. 
![image](https://user-images.githubusercontent.com/44513825/93020942-1b56a800-f5ae-11ea-808f-a01472b977ea.png)